torchao/prototype/mx_formats/custom_cast.py

```diff
@@ -102,6 +122,7 @@ def f6_e3m2_unpacked_to_f32(x: torch.Tensor):
     return _fpx_unpacked_to_f32(x, EBITS_F6_E3M2, MBITS_F6_E3M2)
 
 
+# Note that only E2M1 supports triton currently
```

Comment:
note: all of these kernels support triton via torch.compile, e2m1 is the only one that has a handwritten triton kernel

Reply:
What's motivation for the handwritten triton kernel for e2m1? Is this needed for e3m0 as well?